docs/tutorials/ai/durable-ai-agent/index.md

```diff
@@ -0,0 +1,6078 @@
+
+### Optional
+
+You can opt to use real API services for your tools, or use provided mock functions. 
+
+* A free [RapidAPI Sky Scrapper API Key](https://rapidapi.com/apiheya/api/sky-scrapper) saved securely where you can access it. You will use this to search for flights.
+* A free [Stripe Account](https://stripe.com/lp/start-now) with a configured [sandbox](https://docs.stripe.com/sandboxes). You will use this to generate fake invoices for the flights that are being booked.
+
+### Concepts
+
+Additionally, this tutorial assumes you have basic familiarity with:
+
+#### Programming Concepts
+
+* Temporal fundamentals such as [Workflows](https://docs.temporal.io/develop/python/core-application#develop-workflows), [Activities](https://docs.temporal.io/develop/python/core-application#develop-activities), [Workers](https://docs.temporal.io/develop/python/core-application#run-a-dev-worker), [Signals](https://docs.temporal.io/develop/python/message-passing#signals), and [Queries](https://docs.temporal.io/develop/python/message-passing#queries)
+* Python fundamentals such as functions, classes, async/await syntax, and virtual environments
+* Command line interface and running commands in a terminal or command prompt  
+* REST API concepts including HTTP requests and JSON responses
+* How to set and use environment variables in your operating system
+
+#### AI Concepts
+
+* [A Mental Model for Agentic AI Applications](https://temporal.io/blog/a-mental-model-for-agentic-ai-applications)
+* [Building an agentic system that's actually production ready](https://temporal.io/blog/building-an-agentic-system-thats-actually-production-ready)
+* [Why Agentic Flows Need Distributed Systems](https://temporal.io/blog/from-ai-hype-to-durable-reality-why-agentic-flows-need-distributed-systems)
+
+## Setting up your development environment
+
+Before you start coding, you need to set up your Python developer environment.
+In this step, you will set up your project structure, install the necessary Python packages, and configure the Python environment needed to build your AI agent.
+
+First, create your project using `uv`:
+
+```command
+uv init temporal-ai-agent --python ">=3.9"
+
+```
+
+Next, change directories into your newly created project:
+
+```command
+cd temporal-ai-agent
+```
+
+`uv` is a modern Python project and packaging tool that sets up a project structure for  you.
+Running this command creates the following default Python package structure for you:
+
+```
+temporal-ai-agent/
+├── .gitignore
+├── .python-version
+├── README.md
+├── pyproject.toml
+└── uv.lock
+```
+
+It automatically runs a `git init` command for you, provides you with the default `.gitignore` for Python, creates a `.python-version` file that has the project's default Python version, a README.md, a Hello World `main.py` program, and a `pyproject.toml` file for managing the projects packages and environment.
+
+You won't need the `main.py` file, so delete it:
```

Comment:
This wasn't listed in the above tree which is a bit confusing

Reply:
Good catch. Will add in.